explorer/explorer_test.go

```diff
@@ -26,3 +31,18 @@ func TestSimNetName(t *testing.T) {
 		t.Errorf(`Net name not "Simnet": %s`, netName)
 	}
 }
+
+// func TestThreeSigFigs(t *testing.T) {
```

Comment:
Looks like a really nice change in the screenshots.  Is it a WIP still though?  Maybe just this test?